addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/KNXBridgeHandlerTracker.java

```diff
@@ -0,0 +1,11 @@
+package org.openhab.binding.knx;
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.